test/units/TEST-13-NSPAWN.sh

```diff
@@ -13,6 +13,7 @@ fi
 # shellcheck source=test/units/test-control.sh
 . "$(dirname "$0")"/test-control.sh
 
+export TEST_SKIP_SUBTESTS="nspawn machined"
```

Comment:
I cannot find any failures recently for nspawn. Do you?

Reply:
There was one last week, unfortunately I forgot to save the logs locally and retention is just a few days due to size

Reply:
Hm, OK.